### PR TITLE
logging sig dev code guide

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: developer-guide/concepts-and-designs/sig-observability/kubesphere-monitoring.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+The KubeSphere monitoring system comprises of many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy or create a few components within the namespace `kubesphere-monitoring-system`:
```
Comment (huanggze) [inline developer-guide/concepts-and-designs/sig-observability/kubesphere-monitoring.md]: I think `comprises` is good, or maybe `is comprised of `.

File: developer-guide/concepts-and-designs/sig-observability/kubesphere-logging.md [hunk 2]
```diff
@@ -2,45 +2,45 @@
 
 ## Overview
 
-The KubeSphere logging system comprises many components that work together to achieve centralized log management. Once you enable the logging module during installation, a few components will be deployed or created within the namespace `kubesphere-logging-system`:
+The KubeSphere logging system comprises of many components that work together to achieve unified log management. Once you enable the logging module during installation, a few components will be deployed or created within the namespace `kubesphere-logging-system`:
```
Comment (huanggze) [inline developer-guide/concepts-and-designs/sig-observability/kubesphere-logging.md]: I think `comprises` is good, or maybe `is comprised of `.

File: developer-guide/concepts-and-designs/sig-observability/kubesphere-monitoring.md (hunk 1 above)
Comment (rayzhou2017) [reply developer-guide/concepts-and-designs/sig-observability/kubesphere-monitoring.md]: should be `is comprised of`

**Pull request merged**